app/assets/v2/js/pages/fulfill_bounty/token.js

```diff
@@ -68,3 +71,73 @@ fulfillBounty = data => {
     }
   });
 };
+
+
+const is_valid_address = (address) => {
+  switch (web3_type) {
+
+    // etc
+    // celo
+    // rsk
+
+    case 'binance_ext':
+      if (!address.toLowerCase().startsWith('bnb')) {
```

Comment:
@thelostone-mc binance starts with `0x`

Reply:
forgot to remove that ! thanks for catching that